whylog_vim/output_formater/output_aggregator.py

```diff
@@ -0,0 +1,30 @@
+from whylog_vim.consts import Messages
+
+
+class OutputAggregator(object):
+    def __init__(self):
+        self.buttons = {}
+        self.output_lines = []
+
+    def add(self, element):
+        self.output_lines.append(element)
+
+    def add_commented(self, content):
+        self.add(Messages.COMMENT_PREFIX % content)
+
+    def get_content(self):
+        # The same type as vim buffor.
+        return self.output_lines
+
+    def create_button(self, callback_function, line=None):
```

Comment:
Do you really meant `callback`? Shouldn't it be  `call_function`? or `button_call_function`